content/plugins/aggressive-splitting-plugin.md

```diff
@@ -0,0 +1,23 @@
+---
+title: AggressiveSplittingPlugin
+contributors:
+  - 
+---
+
+

```

Comment:
Can you add a lead-in paragraph here? Just some general details on what this plugin does.